app/views/hyrax/collections/_form_required_information.html.erb

```diff
@@ -20,6 +20,7 @@
 
     <%= render "records/edit_fields/language", f: f, key: 'language' %>
 
+  <%= f.input :doi %>
```

Comment:
It's only for collections. The full form for the identifiers are large while we don't know what will be used except, so I only put doi in the form now, and I think we can just add line here for the identifier we need in the future. See Greg's [#issuecomment-287090601](https://github.com/ucsdlib/horton/issues/62#issuecomment-287090601).